netbout-client/pom.xml

```diff
@@ -189,6 +190,27 @@
                                  Findbugs filters in current version of Qulice
                                 -->
                                 <exclude>findbugs:~com.netbout.client.*</exclude>
+                                <!--
+                                 puzzle #961:30min declared in netbout-web/pom.xml.
```

Comment:
And it should not be. The puzzle is declared in netbout-web/pom.xml and declaring here also will raise puzzle duplication error in travis.